extensions/binding/org.eclipse.smarthome.binding.mqttbroker/src/main/java/org/eclipse/smarthome/binding/mqttbroker/internal/HexUtils.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.binding.mqttbroker.internal;
+
+/**
+ * Provides some byte to Hex-String and Hex to byte operations.
+ *
+ * @author David Graeff - Initial contribution
+ */
+public class HexUtils {
```

Comment:
I assume this utils would make sense to be moved to a core bundle.

Reply:
There is something for hex in apache io actually. But in this specific use case I need to remove white-space and colons between the hex characters.